### PR TITLE
German translations of vocabularies

### PR DESCRIPTION
**Pull request by utnapischtim**
To apply those changes it was necessary to recreate the services because "invenio-cli services setup" only lead to the error: "Errors: Services status inconsistent.Expected False obtained True". 

For the recreation of the services it was necessary to do:

- invenio-cli services stop
- invenio-cli services destroy
- invenio-cli services start
- invenio-cli services setup
- restart the running server

related issue: inveniosoftware/invenio-app-rdm#1306

### CODE REVIEW

**Comment by lnielsen** [comment]
@utnapischtim I think we can merge the PR as is, and fix related issues in later PRs. You you be able to rebase and get green on the tests? Then we can merge. Note #945 is fixing the broken tests on master.

**Comment by zzacharo** [comment]
@utnapischtim you can rebase with latest master now and tests should pass!

File: invenio_rdm_records/fixtures/data/vocabularies/roles.yaml [hunk 1]
```diff
@@ -3,103 +3,124 @@
...
     datacite: RegistrationAuthority
   title:
     en: Registration authority
+    de: Registrierungsbehörde
 - id: relatedperson
   props:
     datacite: RelatedPerson
   title:
     en: Related person
+    de: Verwandte Personen
```
Comment (chriz-uniba) [inline invenio_rdm_records/fixtures/data/vocabularies/roles.yaml:84]: Shouldn't it be "Verwandte Person" -- Singular?

File: invenio_rdm_records/fixtures/data/vocabularies/roles.yaml [hunk 2]
```diff
@@ -3,103 +3,124 @@
...
 - id: researcher
   props:
     datacite: Researcher
   title:
     en: Researcher
+    de: WissenschafterIn
```
Comment (chriz-uniba) [inline invenio_rdm_records/fixtures/data/vocabularies/roles.yaml:90]: "WissenschaftlerIn" (missing an l)

Comment (chriz-uniba) [reply invenio_rdm_records/fixtures/data/vocabularies/roles.yaml:90]: as @utnapischtim mentioned elsewhere: https://www.onlinelektorat.at/1-000-und-1-fakt-aus-der-welt-des-lektorats-wissenschafter/
but as far as I know we are translating de_de, right? So WissenschaftlerIn with the l would be the way to go --- sorry for the Austrian and Swiss community...

**Comment by lnielsen** [comment]
Closed by #1005

**Pull request closed without merging**